cms/static/cms/sass/components/pagetree/_tree.scss

```
@@ -823,20 +875,27 @@
         // we need to add one pixel in to negate the negative margin
         // value to get the borders to work visually
         height: $pagetree-cell-height + 1;
-        padding: $pagetree-anchor-padding;
-        margin-top: -1px;
-        border: 1px solid $gray-lighter;
-        border-right: none;
+        padding-block: $pagetree-anchor-padding-block;
+        padding-inline: $pagetree-anchor-padding-inline;
+        margin-block-start: -1px;
+        border-block: 1px solid $gray-lighter;
+        border-inline: 1px solid $gray-lighter;
+        border-inline-end: none;
         background-color: $white;
         background-image: url('../../img/pagetree/tree-li-drag.gif');
         background-repeat: no-repeat;
-        background-position: -1px center;;
+        background-position: left center;;
         image-rendering: pixelated;
         white-space: normal;
 
         &:hover {
             color: $gray !important;
         }
+
+        [dir=rtl] & {
+            // no direct logical property for background-position
+            background-position: right center;
```

```suggestion
            background-position: calc(100% - 2px) center;
```
A tiny optical improvement...